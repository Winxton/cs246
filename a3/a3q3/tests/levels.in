! alice
! bob
+ bob eve
+ eve fred
! albert
+ bob poop
+ alice jane
* eve
+ bob cool
p
* poop
* cool
? poop
? jane
p
