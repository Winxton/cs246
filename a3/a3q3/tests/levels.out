albert is the parent of bob
bob is the parent of alice,poop,cool
alice is the parent of jane
poop has no children
cool has no children
Not found
albert,bob,alice,jane
albert is the parent of bob
bob is the parent of alice
alice is the parent of jane
